controllers/user.py

```diff
@@ -42,14 +42,14 @@ def get():
             user = get_user_by_email(session, args['email'])
             return user.availabilities
 
-
```

Comment:
Bit of a nitpicky comment, but probably try not to submit style changes in a file that wouldn't otherwise be added, it obscures the valuable changes a little bit.

Reply:
Missed that one, resolved in 9df41e056d3b86e4235f6f8fc845aaa702b03d83